src/assets/drizzle/styles/components/_icon-link.scss

```diff
@@ -0,0 +1,12 @@
+.#{$app-namespace}-c-icon-link {
+  fill: $white;
```

Comment:
Hey Jon! I think for this, we don't need an extra sass file for icon-links. We should set `fill: currentColor;` to the drizzle icons (it's currently missing). And, then if you add the `'drizzle-b-Link drizzle-b-Link--light'` to the link, the color will be set to white and therefore the icon will be white. This reduces code and is more future friendly.

Reply:
Done!